app/flux/API/DashAPI.js

```diff
@@ -0,0 +1,17 @@
+
+var Dash = {
+  getData: (courseName) => {
+    return fetch('/api/course/' + courseName, { method: 'GET' })
+      .then(response => {
+          return response.json();
+      })
+      .then(response => {
```

Comment:
I meant the ones after it, you only need the code block I put in the comment, not the Promise.resolve and catch.